Reverse geocoding and minor fixes

edewaele opened the pull request:
- #31 when the POI lacks address information, a request is sent to Nominatim for retrieving the address
- #47 the popup title is the same as the POI label
- "Locate" was not translated in the address search box

edewaele committed:
Popup title is the same as popup name

For the sake of consistency, when no name is provided, the popup title
may be taken from "ref" or "operator" as in getName()

edewaele committed:
Popup tooltip is the same as popup name

For the sake of consistency, when no name is provided, the popup tooltip
may be taken from "ref" or "operator" as in getName()

edewaele committed:
"Locate" button was not translated

Changed Nomimatim widget so that that it uses the translated string

edewaele committed:
#31 getting addresses from Nominatim

When a POI does not have street name and house number, the address is
obtained from Nominatim

edewaele committed:
French for organic

edewaele committed:
Fixing POI URLs without http:// or https://

Many website keys are not preceded with the protocol identifier, which
must me added in the href

edewaele committed:
"Open OSM" may be translated

This fixes the last untranslated string in issue #8

edewaele commented:
Hey, anyone there ?

nicolashohm commented on nominatim_reverse_proxy.php:
```diff
@@ -0,0 +1,6 @@
+header('Content-Type: application/json');
+$_GET["format"] = "json";
+echo file_get_contents('http://nominatim.openstreetmap.org/reverse?'.http_build_query($_GET));
+?>
```
`?>` is not needed

nicolashohm commented on lang/de_DE.php:
```diff
@@ -348,6 +348,7 @@
 define('JS_LOADING','Lade...');
 define('JS_PLEASE_ZOOM_IN','Bitte hineinzoomen');
 define('JS_OPEN','Offen');
+define('JS_OPEN_OSM','Open in OSM');
```
"Öffnen in OSM" ;)

dotevo commented:
I will push it to test directory today.

dotevo commented:
Why you do not use ajax? Like here https://github.com/dotevo/osm24/blob/master/js/main.js#L288

nicolashohm commented:
@dotevo good point, I also see no reason for using a proxy (I think this is what you mean?)

dotevo commented:
Exactly, I think that can be used but only for Google bot.

edewaele commented:
Even though Webkit authorises such requests, Firefox would not allow a direct ajax request, arguing that this is against "Same-origin" policy.

https://github.com/dotevo/osm24/blob/master/js/main.js#L288 does not work with Firefox either.

edewaele committed:
Added amenity=post_office (fixes issue #37)

edewaele committed:
Added amenity=post_office (fixes issue #37)

Added amenity=post_office (fixes issue #37)

Added amenity=post_office (fixes issue #37)
